fhir-bucket/src/main/java/com/ibm/fhir/bucket/interop/CmsPayerScenario.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bucket.interop;
+
+import java.util.logging.Logger;
+
+import com.ibm.fhir.bucket.client.FhirClient;
+import com.ibm.fhir.model.resource.Bundle;
+
+/**
+ *
+ */
+public class CmsPayerScenario implements IPatientScenario {
```

Comment:
```suggestion
public class InteropPatientScenario implements IPatientScenario {
```

I don't think we should reference CmsPayer

Reply:
ok